From: Simon McVittie <smcv@collabora.com>
Date: Thu, 31 Aug 2017 17:11:59 +0100
Subject: Disable unused support for gobject-introspection

Signed-off-by: Simon McVittie <smcv@collabora.com>
---
 configure.ac | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index a344f75..66ccf52 100644
--- a/configure.ac
+++ b/configure.ac
@@ -170,8 +170,6 @@ if test x$enable_documentation = xyes; then
 fi
 AM_CONDITIONAL(BUILD_DOCUMENTATION, test x$enable_documentation = xyes)
 
-GOBJECT_INTROSPECTION_CHECK([1.40.0])
-
 AC_ARG_ENABLE(docbook-docs,
         [AS_HELP_STRING([--enable-docbook-docs],[build documentation (requires xmlto)])],
         enable_docbook_docs=$enableval, enable_docbook_docs=auto)
