From: Phaedrus Leeds <mwleeds@protonmail.com>
Date: Wed, 17 Nov 2021 14:51:25 -0800
Subject: Fix a typo in a terminal title

Origin: upstream, 1.2.3, commit:c2ec158dc85731b8da6f83db6a2d183f2430f22c
---
 src/builder-manifest.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/builder-manifest.c b/src/builder-manifest.c
index f8450c7..2f95481 100644
--- a/src/builder-manifest.c
+++ b/src/builder-manifest.c
@@ -3673,7 +3673,7 @@ builder_manifest_bundle_sources (BuilderManifest *self,
 
       g_print ("Bundling sources\n");
 
-      builder_set_term_title (_("Bunding sources for %s"), self->id);
+      builder_set_term_title (_("Bundling sources for %s"), self->id);
 
       if (!builder_context_enable_rofiles (context, error))
         return FALSE;
